src/Tools/Source/RunTests/ProcessTestExecutor.cs

```diff
@@ -81,7 +82,16 @@ private string GetResultsFilePath(AssemblyInfo assemblyInfo)
                 // an empty log just in case, so our runner will still fail.
                 File.Create(resultsFilePath).Close();
 
-                var dumpOutputFilePath = Path.Combine(resultsDir, $"{assemblyInfo.DisplayName}.dmp");
+                // Define environment variables for processes started via ProcessRunner.
+                var environmentVariables = new Dictionary<string, string>();
+                environmentVariables.Add(ProcDumpRunner.ProcDumpPathEnvironmentVariableKey, _options.ProcDumpPath);
```

Comment:
I'm not a fan of this approach. Environment variables are essentially global state. Here we're changing global state in order to essentially pass a parameter to a function call. We should just be passing the parameter directly.